lib/engine/round/g_1822/stock.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative '../stock'
+
+module Engine
+  module Round
+    module G1822
+      class Stock < Stock
+        attr_accessor :bids, :bidders
+
+        def finish_round
+          @game.bidbox_minors.map do |minor_company|
+            bid = highest_bid(minor_company)
+            float_minor(bid) unless bid.nil?
+          end
+
+          @game.bidbox_concessions.map do |concessions|
+            bid = highest_bid(concessions)
+            buy_company(bid) unless bid.nil?
+          end
+
+          @game.bidbox_privates.map do |private_company|
+            bid = highest_bid(private_company)
+            buy_company(bid) unless bid.nil?
+          end
+
+          super
+        end
+
+        def setup
```

Comment:
Don't need to override if only calling super.

Reply:
Im miles away before im done with that file, just added because its gonna med populated with code soon.